src/App.js

```diff
@@ -14,11 +15,51 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const preparedGoods = goodsFromServer.map((item, index) => (
+  {
+    id: index,
+    name: item,
+  }
+));
+
+class App extends React.Component {
+  state = {
+    selected: '',
+  }
+
+  clickHandler = (event) => {
+    this.setState({ selected: event.target.textContent });
+  }
+
+  remover = () => {
+    this.setState({ selected: '' });
+  }
```

Comment:
не записывай в одну строку, неудобно поддерживать
```suggestion
  clickHandler = (event) => {
    this.setState({
      selected: event.target.textContent,
    });
  }

  remover = () => {
    this.setState({ selected: '' });
  }
```

Reply:
fixed